cypress/e2e/signIn.cy.js

```diff
@@ -1,34 +1,44 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
-
 import SignInPageObject from '../support/pages/signIn.pageObject';
-import HomePageObject from '../support/pages/home.pageObject';
-
+import HomePageObject from '../support/pages/article.pageObject';
 const signInPage = new SignInPageObject();
 const homePage = new HomePageObject();
-
 describe('Sign In page', () => {
   let user;
-
   before(() => {
     cy.task('db:clear');
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
     });
   });
 
-  it('should provide an ability to log in with existing credentials', () => {
+  beforeEach(() => {
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
-    signInPage.clickSignInBtn();
-
-    homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it.only('should provide an ability to log in with existing credentials',
```

Comment:
The use of `it.only` will cause only this test to run, skipping others. If this is not intentional, consider removing `.only` to ensure all tests are executed.